ecodan-ha-local/ehal_proto.h

```diff
@@ -28,6 +28,7 @@ namespace ehal::hp
 #define SET_SETTINGS_FLAG_HP_MODE 0x08
 #define SET_SETTINGS_FLAG_DHW_MODE 0x04
 #define SET_SETTINGS_FLAG_MODE_TOGGLE 0x1
+#define SET_SETTINGS_FLAG_SYSTEM_MODE_POWER 0x01
```

Comment:
This seems to have the same value as the #define above. Does it need to be it's own define?

Reply:
Yeah, I saw it in the BartGijsber's repository and I thought the same thing. I'll try 0x1 and tell you.